### PR TITLE
Glassdoor

### PR DESCRIPTION
**Pull request by mikedao**
Glassdoor populating.

### CODE REVIEW

File: app/models/company.rb [hunk 1]
```diff
@@ -1,5 +1,27 @@
 class Company < ActiveRecord::Base
+  after_create :get_info
+
   has_many :jobs
 
   validates :name, presence: true
+  private
+
+  def get_info
```
Comment (mikedao) [inline app/models/company.rb:10]: I already refactored enough for emily. Leaving this be.

File: app/models/company.rb [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+    result = GlassdoorService.new.company(self.name)
+    self.update_attributes(url: result["website"],
+                           industry: result["industry"],
+                           logo_url: result["squareLogo"],
+                           overall_rating: result["overallRating"].to_f,
+                           rating_description: result["ratingDescription"],
+                           cultureandvalues_rating: result["cultureAndValuesRating"].to_f,
```
Comment (mikedao) [inline app/models/company.rb]: Leaving lengthy line to preserve variable names.

File: app/models/company.rb [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+                      industry: result["industry"],
+                      logo_url: result["squareLogo"],
+                      overall_rating: result["overallRating"].to_f,
+                      rating_description: result["ratingDescription"],
+                      cultureandvalues_rating: result["cultureAndValuesRating"].to_f,
+                      seniorleadership_rating: result["seniorLeadershipRating"].to_f,
+                      compensationandbenefits_rating: result["compensationAndBenefitsRating"].to_f,
+                      careeropportunities_rating: result["careerOpportunitiesRating"].to_f,
+                      worklifebalance_rating: result["workLifeBalanceRating"].to_f,
+                      recommendtofriends_rating: result["recommendToFriendRating"].to_f,
```
Comment (larsonkonr) [inline app/models/company.rb]: I concur

Comment (EmilyMB) [inline app/models/company.rb]: Maybe shift all these over to two spaces to the right of update_attributes?

File: app/models/company.rb [hunk 4]
```diff
@@ -1,5 +1,28 @@
 class Company < ActiveRecord::Base
+  after_create :get_info
```
Comment (EmilyMB) [inline app/models/company.rb:2]: :+1: on getting this working.

File: app/models/company.rb [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+
+  def get_info
+    result = GlassdoorService.new.company(name)
+    if result
+      update_attributes(url: result["website"],
+                        industry: result["industry"],
+                        logo_url: result["squareLogo"],
+                        overall_rating: result["overallRating"].to_f,
+                        rating_description: result["ratingDescription"],
+                        cultureandvalues_rating: result["cultureAndValuesRating"].to_f,
```
Comment (EmilyMB) [inline app/models/company.rb]: How about moving all these over to two spaces to the right of update_attributes.

Comment (larsonkonr) [inline app/models/company.rb]: I concur

File: app/models/company.rb [hunk 6]
```diff
@@ -1,5 +1,30 @@
...
+                        seniorleadership_rating: result["seniorLeadershipRating"].to_f,
+                        compensationandbenefits_rating: result["compensationAndBenefitsRating"].to_f,
+                        careeropportunities_rating: result["careerOpportunitiesRating"].to_f,
+                        worklifebalance_rating: result["workLifeBalanceRating"].to_f,
+                        recommendtofriends_rating: result["recommendToFriendRating"].to_f,
```
Comment (larsonkonr) [inline app/models/company.rb]: i concur

File: app/models/company.rb [hunk 7]
```diff
@@ -1,5 +1,31 @@
...
+    if result
+      update_attributes(
+        url: result["website"],
+        industry: result["industry"],
+        logo_url: result["squareLogo"],
+        overall_rating: result["overallRating"].to_f,
+        rating_description: result["ratingDescription"],
+        cultureandvalues_rating: result["cultureAndValuesRating"].to_f,
+        seniorleadership_rating: result["seniorLeadershipRating"].to_f,
+        compensationandbenefits_rating: result["compensationAndBenefitsRating"].to_f,
```
Comment (larsonkonr) [inline app/models/company.rb:21]: i concur

**Pull request merged**